### PR TITLE
Fixes issue #52

### PR DESCRIPTION
**Pull request by NigelGreenway**
My previous commit 09eca0e28c7763938a450fa322691fce3d119319 broke an integral part of how Twig is called with just passing an array of paths without namespaces. This fixes that.

:)

### CODE REVIEW

File: tests/TwigTest.php [hunk 1]
```diff
@@ -14,29 +14,55 @@
...
+        $this->assertEquals('Happy Tuesday!', $outputMulti);
+    }
+
     public function testSingleTemplateWithANamespace()
     {
         $views = new Twig([
-            'One' => __DIR__.'/templates',
+            'One' => [
+                __DIR__.'/templates',
+            ],
```
Comment (akrabat) [inline tests/TwigTest.php]: Why have you changed this test and not added another one?

Comment (NigelGreenway) [inline tests/TwigTest.php]: Hi @akrabat,

Are we talking about `testSingleTemplateWithANamespace`?

Comment (akrabat) [inline tests/TwigTest.php]: Yes. Why are we no longer testing

```php
$views = new Twig([
    'One' => __DIR__.'/templates',
]);
```

Comment (NigelGreenway) [inline tests/TwigTest.php]: Right, see it. Will sort it in the next hour; didn't notice that change.

**Pull request merged**